src/index.ml

```diff
@@ -338,8 +338,9 @@ module Make (K : Key) (V : Value) (IO : IO) = struct
   (* Merge [log] with [t] into [dst_io]. [log] must be sorted by key hashes. *)
   let merge_with log index dst_io =
     let entries = 10_000 in
```

Comment:
I know that you did not change this, but why are the number of entries 10,000?

Reply:
We want to avoid reading all the entries one by one since this is more costly, so we read them 10_000 by 10_000 so that it is faster.

Reply:
Ahhh I see, why specifically 10,000 though? Just curious :D

Reply:
No specific reason, it seemed okay during the tests/benchs we had.